lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -812,6 +822,14 @@ def test_bypass_course_info(self):
         response = views.course_info(request, course_id)
         self.assertEqual(response.status_code, 200)
 
+    def test_accordion(self):
+        view = views.render_accordion(
+            self.request.user, self.request, self.course,
+            unicode(self.course.get_children()[0].scope_ids.usage_id), None, None
+        )
+        self.assertIn('href="/courses/org.0/course_0/te%EA%9C%B1%E1%B4%9B_course/courseware/Chapter_1/Sequential_1/"', view)
```

Comment:
How does this assertion verify that the correct target_url is associated with its corresponding target_name?  I thought we needed the unit test to verify this binding so that bokchoy test doesn't need to cover it.